Makefile.e2e

```diff
@@ -46,7 +46,7 @@ test-e2e: config-sync-manifest-local __install-nomos-local test-e2e-nobuild
 # Run the Go e2e tests without building images/manifests.
 # Useful for modifying test code and rerunning tests without rebuilding images.
 .PHONY: test-e2e-go-nobuild
-test-e2e-nobuild: "$(KUSTOMIZE)" "$(HELM)" "$(CRANE)"
+test-e2e-nobuild: "$(KUSTOMIZE)" "$(HELM)" "$(CRANE)" "$(COSIGN)"
```

Comment:
If you're choosing to download this at runtime, it doesn't need to also be installed by the Dockerfile into the image

Reply:
Removed

Reply:
Seems like it's still needed for the kind presubmit

Reply:
It can be added to `build-kind-e2e` below to consistently download at runtime

Reply:
Yeah just did, see if it works

Reply:
And it did :]

Reply:
It's still being installed in the Dockerfile, I'd move it to the `build-kind-e2e` target below for consistency.

Reply:
Done